website/docs/faq.md

```diff
@@ -132,6 +132,16 @@ Yes! You can export and download your current product as a standard JSON file an
 - All tags
 - All segments
 
+### How long do you keep my data?
+
+We keep organization data as long as we see activity in that organization.
+After several months of inactivity, we send you a series of email notifications about or plans to delete your organization and all associated data.
+We consider an organization inactive if it meets all of the following criteria:
+- no audit log events are generated in the organization,
+- no valid REST API calls are made to the organization,
```

Comment:
ez melyik API? Rest apirol mashol nem irunk. van dashboard api es public mgmt api.

Reply:
beírtam most, hogy Public Management API